cmd/kubeadm/app/api/types.go

```diff
@@ -86,15 +88,10 @@ var (
 		"rackspace",
 		"vsphere",
 	}
-	SupportedCloudProviders map[string]bool
 )
 
 func init() {
 	_, DefaultServicesCIDR, _ = net.ParseCIDR(DefaultServicesCIDRString)
```

Comment:
What is the motivation for this change below? Is it that it allows the user to have the message in https://github.com/errordeveloper/kubernetes/pull/17/files#r80388324 printed?